sagemaker-featurestore/feature_store_client_side_encryption.ipynb

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
<p>Any reasons that we cannot encrypt the identifier? Even if it is encrypted, it is still unique right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Yes it can be encrypted. I made this update.